docs/DavidTereshchenko/calculator.html

```diff
@@ -0,0 +1,120 @@
+                    <div class="header__logo">
+                        <img src="img/logo.png" alt="logo" width="55" height="58">
+                    </div>
+                    <div class="header__body">
+                        <nav class="header__menu menu">
+                            <ul class="menu__list">
+                                <li class="menu__item">
+                                    <a href="calculator.html" class="menu__link">Calculator</a>
+                                </li>
+                                <li class="menu__item">
+                                    <a href="game.html" class="menu__link">Game</a>
+                                </li>
+                                <li class="menu__item">
+                                    <a href="index.html" class="menu__link">About</a>
+                                </li>
+                                <li class="menu__item">
+                                    <a href="login.html" class="menu__link">Login</a>
+                                </li>
+                            </ul>
+                        </nav>
+                    </div>
+                    <div class="header__burger">
+                        <span></span>
+                    </div>
+                </div>
+            </header>
+            <main class="page">
+                <div class="main__container">
+                    <div class="title__calculator">
+                        <h1>Calculate Games</h1>
+                    </div>
+                    <div class="calculator__container">
+                        <div class="number__container">
+                            <div class="number-1">
+                                <label for="firstNumber">First number</label>
+                                <input id="firstNumber" data-number-one="one" class="number__block one" name="number1" required>
+                            </div>
+                            <div class="operator__block">
+                                <label>
+                                    <select class="operator" data-operation="operator">
+                                        <option value="+">+</option>
+                                        <option value="-">-</option>
+                                        <option value="×">×</option>
+                                        <option value="/">/</option>
+                                        <option value="%">%</option>
+                                        <option value="**">**</option>
+                                    </select>
+                                </label>
+                            </div>
+                            <div class="number-2">
+                                <label for="secondNumber">Second number</label>
+                                <input id="secondNumber" data-number-two="two" class="number__block two" name="number2" required>
+                            </div>
```

Comment:
The same story:
You can remove `<div class="number-2">` and simplify your code:

```
<label for="secondNumber">First number
      <input id="secondNumber" data-number-two ="two" class="number__block one" name="number2" required>
</label>
```